addons/mail/static/src/messaging/component/partner_mention_suggestion/partner_mention_suggestion.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.messaging.component.PartnerMentionSuggestion" owl="1">
+        <t t-if="partner and composer">
+            <t t-if="partner.divider">
+                <div class="dropdown-divider" role="separator"/>
+            </t>
```

Comment:
`divider` should not be on partner, this is not a generic partner data, it's contextual information about the suggestion list.

Also, the separator should not be on this file that renders a single line, it should be on the list itself.

Reply:
Will be handled when relevant partners system will be developed

Reply:
Done